src/integrationTest/groovy/wooga/gradle/IntegrationSpec.groovy

```diff
@@ -46,7 +47,15 @@ class IntegrationSpec extends nebula.test.IntegrationSpec {
     }
 
     Boolean fileExists(String... path) {
-        fileExists(path.join("/"))
+        fileExists(path.join(File.separator))
```

Comment:
If you need the array API can you at least use the utility method internally? Or can this go now?
```suggestion
        FileUtils.getFile(path).exists
```

Reply:
Yes, I will have it used the utility method internally.